signal_scanner_bot/env.py

```diff
@@ -17,22 +17,42 @@ class _State:
     STOP_REQUESTED = False
 
 
-def _env(key: str, fail: bool = True, default: Any = None) -> Optional[str]:
+def _env(
+    key: str,
+    convert: Callable[[str], Any] = None,
+    fail: bool = True,
+    default: Any = None,
+) -> Optional[Any]:
     value = os.environ.get(key)
     if value is None:
         if fail and default is None:
             raise KeyError(f"Key '{key}' is not present in environment!")
         return default
+    if convert is not None:
+        value = convert(value)
     _VARS.append((key, value))
     return value
 
 
+def _cast_to_bool(to_cast: str) -> bool:
+    return to_cast.lower() == "true"
+
+
+def _cast_to_list(to_cast: str) -> List[str]:
+    return to_cast.split(",")
+
+
+def _cast_to_set(to_cast: str) -> Set[str]:
+    return set(to_cast.split(","))
```

Comment:
Definitely your call, but this could be:
```suggestion
    return set(_cast_to_list(to_cast))
```

Reply:
I dunno why but I like this better so I'm just gonna go with it.